src/lib.rs

```diff
@@ -104,24 +104,24 @@ pub enum Error {
 impl Error {
 
     /// Create a nix Error from a given errno
-    pub fn from_errno(errno: errno::Errno) -> Error {
+    pub fn from_errno(errno: Errno) -> Error {
         Error::Sys(errno)
     }
 
     /// Get the current errno and convert it to a nix Error
     pub fn last() -> Error {
-        Error::Sys(errno::Errno::last())
+        Error::Sys(Errno::last())
     }
 
     /// Create a new invalid argument error (`EINVAL`)
     pub fn invalid_argument() -> Error {
-        Error::Sys(errno::EINVAL)
+        Error::Sys(Errno::EINVAL)
```

Comment:
I'm not certain that I like `Errno::EINVAL` and `Errno::last()`. I don't think it makes sense for `last()` to be in the same place in the heirarchy as `EINVAL`.